### PR TITLE
SolitareStation FIXES some access problems

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

chaplains can access morgue
chaplains and curators can access service maints and service lathe
Moved 2 pipes 1 tile up to connect loose ends
## Why It's Good For The Game

M

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
expansion: Expands content of an existing feature
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by HugoOdaX** [comment]
Fix your changelog or else....😡

File: _maps/map_files/SolitaireStation/Solitairestation.dmm [hunk 1]
```diff
@@ -43779,7 +43781,8 @@
 "mku" = (
 /obj/structure/cable,
 /obj/machinery/door/airlock/maintenance{
-	req_access_txt = "12"
+	name = "Service Maintenance Access";
+	req_one_access_txt = "12;22;25;26;28;35;37;38;46;70"
```
Comment (SgtHunk) [inline _maps/map_files/SolitaireStation/Solitairestation.dmm:43785]: this is a clusterfuck. Change it to service.

File: _maps/map_files/SolitaireStation/Solitairestation.dmm [hunk 2]
```diff
@@ -46675,7 +46678,8 @@
 /area/space)
 "nbV" = (
 /obj/machinery/door/airlock/maintenance{
-	req_access_txt = "12"
+	name = "Service Maintenance Access";
+	req_one_access_txt = "12;22;25;26;28;35;37;38;46;70"
```
Comment (SgtHunk) [inline _maps/map_files/SolitaireStation/Solitairestation.dmm:46682]: read above

File: _maps/map_files/SolitaireStation/Solitairestation.dmm [hunk 3]
```diff
@@ -66648,7 +66650,8 @@
 /area/command/heads_quarters/hop)
 "tqQ" = (
 /obj/machinery/door/airlock/maintenance{
-	req_access_txt = "12"
+	name = "Service Maintenance Access";
+	req_one_access_txt = "12;22;25;26;28;35;37;38;46;70"
```
Comment (SgtHunk) [inline _maps/map_files/SolitaireStation/Solitairestation.dmm:66654]: read above

**Review by SgtHunk — CHANGES_REQUESTED** [review]
they made service access and u do this :(

File: _maps/map_files/SolitaireStation/Solitairestation.dmm (hunk 3 above)
Comment (SmoSmoSmoSmok) [reply _maps/map_files/SolitaireStation/Solitairestation.dmm:66654]: > read above

ASSHOLE i didnt do anything i copy pasted the service door on helio and switched it with the maint door on solitare

Comment (SmoSmoSmoSmok) [reply _maps/map_files/SolitaireStation/Solitairestation.dmm:66654]: https://prnt.sc/26maugs

Comment (SgtHunk) [reply _maps/map_files/SolitaireStation/Solitairestation.dmm:66654]: "i used a bad station and the design was bad" cry about it liberal

**Comment by SmoSmoSmoSmok** [comment]
I deleted this branch like a week ago SO i cant resolve conflicts anymore or somth

**Pull request closed without merging**